sass/_component_toc.scss

```diff
@@ -2,26 +2,30 @@
 .toc {
     a {
         position: relative;
-        display: block;
-        color: inherit;
+        display: inline-block;
         line-height: 1.25;
         padding: ($spacer / 4) 0;
```

Comment:
I think padding should be `0`. As items are too big. 

Is it designed this way? Than we should ignore that 'suggestion' ;)

Reply:
I have reduced the toc item spacing in b26efea. I did not completely remove it, as I think we should try to aim for a compromise between use of screen estate and mobile accessibility / design.